layouts/default.vue

```diff
@@ -120,12 +120,14 @@
 </style>
 
 <script>
+import BackgroundImage from '~/assets/images/dega-default-image.png';
 export default {
   data() {
     return {
       toggleNavBar: false,
       toggleMore: true,
-      organisation: Object
+      organisation: Object,
+      prodBaseUrl: process.env.domainHostname
```

Comment:
This needs to be picked from organization entity

Reply:
https://github.com/factly/dega-gateway/issues/186